aviary/examples/external_subsystems/simple_weight/simple_weight.py

```diff
@@ -16,6 +16,7 @@
 
 
 class SimpleWeight(om.ExplicitComponent):
+    """A simple weight component that computes wing mass."""
```

Comment:
This component is pretty fictional (wing mass being 1.5 times the engine mass is nonsensical).  Change to:

"A simple component that compute a wing mass as a function of the engine mass. That values are not representative of any existing aircraft, and the component is meant to demonstrate the concept of an externally calculated subsystem mass."

Reply:
Done.